docs/scalardb-supported-databases.md

```diff
@@ -1,31 +1,141 @@
+|:------------------|:---------------|:-----------------|:----------------|:-----------------|:----------------|
+| **ScalarDB 3.10** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ❌ | ❌ | ✅ | ✅ | ✅ |
+
+## Azure Cosmos DB for NoSQL
+
+| Version | Azure Cosmos DB for NoSQL |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## JDBC databases
+
+{% capture notice--info %}
+**Note**
+
+For recommendations when using JDBC databases, see [JDBC database recommendations](requirements/#jdbc-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Amazon Aurora MySQL
+
+| Version | Amazon Aurora MySQL 3 | Amazon Aurora MySQL 2 |
+:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### Amazon Aurora PostgreSQL
+
+| Version | Amazon Aurora PostgreSQL 14 | Amazon Aurora PostgreSQL 13 | Amazon Aurora PostgreSQL 12 |
+:---------------|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ | ✅ |
+
+### Microsoft SQL Server
+
+| Version | Microsoft SQL Server 2022 | Microsoft SQL Server 2019 | Microsoft SQL Server 2017 |
+:---------------|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ | ✅ |
+
+### MySQL
+
+| Version | MySQL 8.0 | MySQL 5.7 |
+|:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### Oracle
+
+| Version | Oracle 21.3.0-xe | Oracle 18.4.0-xe |
+:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### PostgreSQL
+
+| Version | PostgreSQL 14 | PostgreSQL 13 | PostgreSQL 12 |
```

Comment:
@brfrn169 Based on the comment in https://github.com/scalar-labs/scalardb/pull/1012#discussion_r1302373516, I think we should add PostreSQL 15 to this table. 

Do ScalarDB versions 3.4 to 3.10 all support PostreSQL 15?

Reply:
@josh-wong 

@Torch3333 is working on the database-compatibility testing to check that. Do you want to wait for it before merging this PR? Or do you want to merge it with with the current databases and versions and update it after finishing the test?

Reply:
@brfrn169 Ah, I see. I assumed, since the PR was merged, that those PostgreSQL and Oracle 23c were confirmed to be compatible.

No need to wait for database-compatibility testing to be finished. We can proceed with merging this PR and update later as needed👍